konnectintech/Konectin-website-_backend
# Internship fix

# Pull Request: Fix Internship Application Endpoint

## Description
Updated the internship application endpoint that allows users to submit their applications with basic details and resume files.

## Changes Made
1. Updated internship application controller with file upload handling and relevant required fields (some fields not present on the design have been added because they are required in the internship schema)

### POST /internshipApplication
Submit a new internship application

## Testing Instructions
1. Use Postman to test the endpoint
2. Set request type to POST
3. Use form-data to include file uploads
4. Ensure all required fields are included with valid enum values



@Michaelajayi150
